os/arch/arm/src/s5j/s5j_boot.c

```diff
@@ -124,7 +124,7 @@ void up_copyvectorblock(void)
 {
 	uint32_t *src = (uint32_t *)&_vector_start;
 	uint32_t *end = (uint32_t *)&_vector_end;
-	uint32_t *dest = (uint32_t *)VECTOR_BASE;
+	uint32_t *dest = (uint32_t *) VECTOR_BASE;
```

Comment:
remove space here.
This file also previous is right except call s5jclkinit instead of cal_init.
@btheosam please check this.

Reply:
done